notebooks/20_markov_decision_processes_part_2/index.md

```diff
@@ -0,0 +1,672 @@
+<div style="direction:ltr;line-height:300%;">
+	<font face="XB Niloofar" size=20>
+        Markov Decision (from value iteration)
+    </font>
+    </div>
+
+<br/>
+
+<br/>
+
+<h1> Table of Contents </h1>
+---
+
+- [Introduction](#introduction)
+- [Value Iteration](#value_iteration)
+    - [Convergence of Value Iteration](#vi_convergence)
+        - [Contraction](#vi_cont)
+        - [Error Bound ](#vi_error)
+    - [Value Iteration pseudocode](#vi_code)
+        - [Time Complexity](#time_vi)
+- [Policy Iteration](#pi)
+    - [The idea of policy iteration](#pi_id)
+    - [Policy Evaluation](#pe)
+        - [Fixed Policy](#fp)
+    - [Policy Extraction (Improvement)](#pex)
+        - [Computing Actions from Values](#pi_cafv)
+        - [Computing Actions from Q-Values](#pi_cafq)
+    - [Policy Iteration Summary](#pis)
+    - [Policy Iteration pseudocode](#pi_code)
+- [Conclusion](#conclusion)
+- [References](#references)
+
+<br/>
+
+<div id='introduction'><h1> Introduction </h1></div>
+---
+
+<br/>
+
+In the previous lecture, you became familiar with what MDP is. As you know, MDP can be remembered as a mathematical framework used for modeling decision-making problems. Usually, an agent can do some actions, but the outcomes are stochastic and not entirely controllable. The goal is to decide the best action to select based on his current state. It’s better to know MDP before Reinforcement Learning.
```

Comment:
"As you know might not be a proper phrase here. Try using more formal phrases.
"His" should be replaced by its in my opinion.
In scientific articles, it is better not to use contractions (it is instead of it's)